nilearn/datasets/description/dosenbach_2010.rst

```diff
@@ -0,0 +1,22 @@
+Dosenbach 2010 atlas
+
+
+Notes
+-----
+160 ROIs covering much of the cerebral cortex and cerebellum.
+They were obtained from meta-analyses of fMRI activation studies
+and assigned into 6 networks according to a modularity analysis of
```

Comment:
"modularity analysis" does not mean anything -> perhaps add a half sentence that at least is not ambiuous about what will be fetched